lib/FindStack.cmake

```diff
@@ -11,7 +11,7 @@ IF (STACK_EXECUTABLE)
                     ERROR_QUIET)
     STRING(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" GHC_VERSION "${_GHCVERNO}")
 
-    IF ("${GHC_VERSION}" VERSION_EQUAL "7.8.4")
+    IF ("${GHC_VERSION}" VERSION_EQUAL "7.10.2")
```

Comment:
@LordMord Why do you need that? There wouldn't be anything wrong with this, except for the fact that ocpkg installs GHC 7.8.4 https://github.com/opencog/ocpkg/blob/master/ocpkg#L166

So, if you really want to go with the upgrade (cause say you're using some new features of GHC 7.10.2) you would have to have take care of ocpkg as well.